mindmeld/augmentation.py

```diff
@@ -15,19 +15,33 @@
 
 import logging
 import re
+import string
+import random
+import os
+import zipfile
 
 from abc import ABC, abstractmethod
 from tqdm import tqdm
+from urllib.request import urlretrieve
+
 
 from ._util import get_pattern, read_path_queries, write_to_file
 from .components._util import _is_module_available, _get_module_or_attr
 from .models.helpers import register_augmentor, AUGMENTATION_MAP
+from .markup import load_query, dump_query
+from .core import Entity, Span, QueryEntity, ProcessedQuery, _get_overlap
+from .path import EMBEDDINGS_FOLDER_PATH, PARAPHRASER_FILE_PATH, PARAPHRASER_MODEL_PATH
+from .models.taggers.embeddings import TqdmUpTo
 
 logger = logging.getLogger(__name__)
 
 # pylint: disable=R0201
 
 SUPPORTED_LANGUAGE_CODES = ["en", "es", "fr", "it", "pt", "ro"]
+EOS_TOKEN = '</s>'
+NUM_PARAPHRASES = 10
```

Comment:
Nit: Consider renaming to `DEFAULT_NUM_PARAPHRASES` for clarity

Reply:
Also, consider adding "EN" since this number applies only to that model.